doc/source/data/api/dataset.rst

```diff
@@ -0,0 +1,285 @@
+.. _dataset-api:
+
+Dataset API
+===========
+
+.. autoclass:: ray.data.Dataset
+
+**Basic Transformations**
+
+.. autosummary::
+    :nosignatures:
+
+    ray.data.Dataset.map
+    ray.data.Dataset.map_batches
+    ray.data.Dataset.flat_map
+    ray.data.Dataset.filter
+    ray.data.Dataset.add_column
+    ray.data.Dataset.drop_columns
+    ray.data.Dataset.random_sample
+    ray.data.Dataset.limit
+
+**Sorting, Shuffling, Repartitioning**
+
+.. autosummary::
+    :nosignatures:
+
+    ray.data.Dataset.sort
+    ray.data.Dataset.random_shuffle
+    ray.data.Dataset.randomize_block_order
+    ray.data.Dataset.repartition
+
+**Splitting and Merging Datasets**
+
+.. autosummary::
+    :nosignatures:
+
+    ray.data.Dataset.split
+    ray.data.Dataset.split_at_indices
+    ray.data.Dataset.split_proportionately
+    ray.data.Dataset.train_test_split
+    ray.data.Dataset.union
+    ray.data.Dataset.zip
+
+**Grouped and Global Aggregations**
+
+.. autosummary::
+    :nosignatures:
+
+    ray.data.Dataset.groupby
+    ray.data.Dataset.aggregate
+    ray.data.Dataset.sum
+    ray.data.Dataset.min
+    ray.data.Dataset.max
+    ray.data.Dataset.mean
+    ray.data.Dataset.std
+
+**Converting to Pipeline**
+
+.. autosummary::
+    :nosignatures:
+
+    ray.data.Dataset.repeat
+    ray.data.Dataset.window
+
+**Consuming Datasets**
+
+.. autosummary::
+    :nosignatures:
+
+    ray.data.Dataset.show
+    ray.data.Dataset.take
+    ray.data.Dataset.take_all
+    ray.data.Dataset.iter_rows
+    ray.data.Dataset.iter_batches
+    ray.data.Dataset.iter_torch_batches
+    ray.data.Dataset.iter_tf_batches
+
+**I/O and Conversion**
+
+.. autosummary::
+    :nosignatures:
+
+    ray.data.Dataset.write_parquet
```

Comment:
why we don't list any `read_xxx` APIs here?

Reply:
This page is only for `ray.data.Dataset`.